examples/docs_beta_snippets/docs_beta_snippets/guides/components/shell-script-component/2-shell-command-empty.py

```diff
@@ -2,14 +2,13 @@
 from dagster_components import (
     Component,
     ComponentLoadContext,
-    DefaultComponentScaffolder,
     ResolvableModel,
 )
 
-class ShellCommandSchema(ResolvableModel):
+class ShellCommandModel(ResolvableModel):
```

Comment:
should this class even be present at this point since the below class is a model component

Reply:
I was going to redo this once we refresh the docs. It might be there for a reason in context.